apps/passport-client/components/screens/MultiChoiceScanScreen.tsx

```diff
@@ -0,0 +1,219 @@
+import { useCallback, useState } from "react";
+import { QrReader } from "react-qr-reader";
+import { useNavigate } from "react-router-dom";
+import { SingleValue } from "react-select";
+import styled from "styled-components";
+import { useDispatch, useLaserScannerKeystrokeInput } from "../../src/appHooks";
+import { loadUsingLaserScanner } from "../../src/localstorage";
+import { maybeRedirect } from "../../src/util";
+import { H5, Spacer, TextCenter } from "../core";
+import { ScanditScanner } from "../core/Scandit";
+import Scanner from "../core/Scanner";
```

Comment:
generally avoid default exports, we could make this be an error rather than a warning